modules/rosa-hcp-vpc-manual.adoc

```diff
@@ -19,8 +19,8 @@ To manually create your Virtual Private Cloud (VPC), go to link:https://us-east-
 | CIDR range
 | Your VPC CIDR range should match your machine CIDR.
 
-| Availability Zone
-| You need 1 Availability Zone for a single zone, and you need 3 for Availability Zones for multi-zone.
+| Availability zone
+| You need 1 availability zone for a single zone, and you need 3 for availability zones for multi-zone.
```

Comment:
Above, you write out "one" and "seven." Here, you're using numerals. Not sure if you want consistency (https://www.ibm.com/docs/en/ibm-style?topic=measurement-numerals-versus-words) if it's it's OK since they're in different sections. No preference either way, just pointing it out.